tools/clang/lib/SPIRV/SpirvBuilder.cpp

```diff
@@ -1666,14 +1666,22 @@ SpirvConstant *SpirvBuilder::getConstantNull(QualType type) {
 
 SpirvString *SpirvBuilder::getString(llvm::StringRef str) {
```

Comment:
Nit: This is minor issue, but now this method becomes long .. changing the method name to `SpirvBuilder::getOrCreateString(..)` would be nice.
In addition, I'd like to separate the part to create the string to another method i.e.,
```
SpirvString *SpirvBuilder::createString(llvm::StringRef str) {
  // Create a SpirvString instruction
  auto *instr = new (context) SpirvString(/* SourceLocation */ {}, str);
  instr->setRValue();
  stringLiterals[str.str()] = instr;
  if (str.empty())
    emptyString = instr;
  else
    stringLiterals[str.str()] = instr;
  mod->addString(instr);
  return instr;
}
```

Reply:
Done.